File: conditionals/time_of_day_notes.py
# Josephine Chiu, How to get the time of day for Python

import time
#print(time.gmtime(0)) #gmtime(0) first instance of time in ptrogramming

#current time in seconds since gmtime
current =time.time()

#Current time as we are used to seeing it
now = time.ctime(current)
print(now)

#get just the hour
local_time = time.localtime(current)
hours = local_time.tm_hour
minutes = local_time.tm_min
print(hours)
print(minutes)

File: conditionals/time_of_day_notes.c
// Josephine Chiu, How to get the time of day for C
#include <stdio.h>
#include <time.h>

int main(void){
    // time since Jan 1 1970
    time_t seconds;
    seconds = time(NULL);
    //printf("Seconds since January 1, 1970 = %d\n", seconds);
    
    //Current time
    time_t rawtime;
    struct tm*timeinfo;
    time(&rawtime);
    timeinfo = localtime(&rawtime);
    printf("Current time and date is %s", asctime(timeinfo));

    //Current hour
    time_t now = time(NULL);
    struct tm*tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);

    return 0;
}